### PR TITLE
chore: Upgrade apps to latest node lts

### PR DESCRIPTION
**Pull request by ivo-toby**
## Purpose
Update the node version of lambda packages in this mono-repo to release 16

## Dependencies  & package versions
I've updated packages to match the peerdependencies, because Lerna was not able to run successfully.
I have also updated `@contentful/app-scripts` because the <1 releases have a npm/node-version restriction and that would not run as well. 

### Note on typesafety
`smartling/lambda` is partially typed, but not entirely. Since `serverless-express` lacks ts definitions and express itself is quite old it is not really viable to introduce typings to this package as of now. Hence the `@ts-ignore` statements

### CODE REVIEW

File: apps/jira/functions/package.json [hunk 1]
```diff
@@ -3,16 +3,6 @@
   "version": "1.4.62",
   "main": "index.js",
   "private": true,
-  "scripts": {
```
Comment (kdamball) [inline apps/jira/functions/package.json:6]: Why is this deleted?

**Review by kdamball — COMMENTED** [review]
Can we also update netlify lambda to v16 for node

File: apps/jira/functions/package.json (hunk 1 above)
Comment (ivo-toby) [reply apps/jira/functions/package.json]: I suspect a rebase to master, I've reinstated the script-section.

File: apps/graphql-playground/package.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "version": "0.2.5",
   "private": true,
   "devDependencies": {
-    "@contentful/app-scripts": "^0.15.8",
+    "@contentful/app-scripts": "^1.2.0",
```
Comment (colomolo) [inline apps/graphql-playground/package.json:6]: Should we also pin this one, similar to other apps?

**Review by colomolo — COMMENTED** [review]
Overall looks good to me

File: apps/graphql-playground/package.json (hunk 2 above)
Comment (ivo-toby) [reply apps/graphql-playground/package.json]: Done!

**Review by colomolo — APPROVED** [review]
LGTM, but maybe we need another pair of 👀 to look just in case.

File: apps/smartling/lambda/src/index.ts [hunk 3]
```diff
@@ -17,6 +17,7 @@ function requestProxier(proxyUrl: string, proxyPathPrefix = '') {
 export function makeApp(fetchFn: any, issuer: any) {
   const app = express() as Express;
 
+  // tslint:disable-next-line
```
Comment (kdamball) [inline apps/smartling/lambda/src/index.ts:20]: Would be great to know why these are now needed. Feel free to add a comment

Comment (ivo-toby) [reply apps/smartling/lambda/src/index.ts]: I fixed the issue tslint was nagging about. 
The rest of this file could need some TLC eventually and we should consider replacing TSLint with ESLint.

**Review by kdamball — APPROVED** [review]
